src/Installer/pkg/dotnet-sdk.csproj

```diff
@@ -0,0 +1,127 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
```

Comment:
tiny nit: please add an empty line here and at the end before the project closing tag for readability.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/45047/commits/0301e6b08a90ad8bcddcbd58ee44ed12fcd39d17